docs/preview_features/build.md

```diff
@@ -0,0 +1,115 @@
+
+## Introduction
+Sometimes you might need to build packages from source. This can be due to a variety of reasons, such as:
+
+1. You are using the package for local development, and want to install and build the package into your workspace.
+2. You want to use source for a package that is not available on conda-forge.
+3. You want to build your package locally into a conda package.
+
+We've been working to support these use-cases with the `build` feature in pixi. 
+The vision is to enable building of packages from source, for any language, on any platform.
+
+??? note "Preview feature"
+    This feature is currently in preview phase. To enable use the [preview feature](../reference/pixi_manifest.md#preview-features).
+    ```toml
+    [project]
+    # .. other configuration
+    preview = ["build"]
```

Comment:
```suggestion
    preview = ["pixi-build"]
```
Maybe good to add these code examples to the manifests in `docs/source_files/pixi_tomls` So we automatically test them in ci.

Reply:
Does that work for partial tomls though?